concepticondata/conceptlists/Hale-1846-179.tsv

```diff
@@ -0,0 +1,180 @@
+Hale-1846-179-52	52	Sky; Heaven		
+Hale-1846-179-53	53	Sun	1343	SUN
+Hale-1846-179-54	54	Moon	1313	MOON
+Hale-1846-179-55	55	Star	1430	STAR
+Hale-1846-179-56	56	Day	1225	DAY (NOT NIGHT)
+Hale-1846-179-57	57	Night	1233	NIGHT
+Hale-1846-179-58	58	Light	630	LIGHT (RADIATION)
+Hale-1846-179-59	59	Darkness	11	DARKNESS
+Hale-1846-179-60	60	Morning	1339	MORNING
+Hale-1846-179-61	61	Evening	1629	EVENING
+Hale-1846-179-62	62	Spring	1537	SPRINGTIME
+Hale-1846-179-63	63	Summer	1342	SUMMER
+Hale-1846-179-64	64	Autumn	1276	AUTUMN
+Hale-1846-179-65	65	Winter	1471	WINTER
+Hale-1846-179-66	66	Wind	960	WIND
+Hale-1846-179-67	67	Thunder	1150	THUNDER
+Hale-1846-179-68	68	Lightning	1654	LIGHTNING
+Hale-1846-179-69	69	Rain	658	RAIN (PRECIPITATION)
+Hale-1846-179-70	70	Snow	784	SNOW
+Hale-1846-179-71	71	Hail	609	HAIL
+Hale-1846-179-72	72	Fire	221	FIRE
+Hale-1846-179-73	73	Water	948	WATER
+Hale-1846-179-74	74	Ice	617	ICE
+Hale-1846-179-75	75	Earth; Land	3603	EARTH OR LAND
+Hale-1846-179-76	76	Sea	1474	SEA
+Hale-1846-179-77	77	River	666	RIVER
+Hale-1846-179-78	78	Lake	624	LAKE
+Hale-1846-179-79	79	Valley	921	VALLEY
+Hale-1846-179-80	80	Hill; Mountain	2118	MOUNTAIN OR HILL
+Hale-1846-179-81	81	Island	622	ISLAND
+Hale-1846-179-82	82	Stone	857	STONE
+Hale-1846-179-83	83	Salt	1274	SALT
+Hale-1846-179-84	84	Iron	621	IRON
+Hale-1846-179-85	85	Tree	906	TREE
+Hale-1846-179-86	86	Wood	1803	WOOD
+Hale-1846-179-87	87	Leaf	628	LEAF
+Hale-1846-179-88	88	Bark	1204	BARK
+Hale-1846-179-89	89	Grass	606	GRASS
+Hale-1846-179-90	90	Pine	1740	PINE
+Hale-1846-179-91	91	Flesh; Meat	2615	FLESH OR MEAT
+Hale-1846-179-92	92	Dog	2009	DOG
+Hale-1846-179-93	93	Buffalo	934	BUFFALO
+Hale-1846-179-94	94	Bear	1820	BEAR
+Hale-1846-179-95	95	Wolf	522	WOLF
+Hale-1846-179-96	96	Deer	1936	DEER
+Hale-1846-179-97	97	Elk	1772	ELK
+Hale-1846-179-98	98	Beaver	1194	BEAVER
+Hale-1846-179-99	99	Tortoise	895	TORTOISE
+Hale-1846-179-100	100	Fly	1504	FLY (INSECT)
+Hale-1846-179-101	101	Musquito	1509	MOSQUITO
+Hale-1846-179-102	102	Snake	730	SNAKE
+Hale-1846-179-103	103	Bird	937	BIRD
+Hale-1846-179-104	104	Egg	744	EGG
+Hale-1846-179-105	105	Feathers	1201	FEATHER
+Hale-1846-179-106	106	Wings	1257	WING
+Hale-1846-179-107	107	Duck	1360	DUCK
+Hale-1846-179-108	108	Pigeon	1853	DOVE
+Hale-1846-179-109	109	Fish	227	FISH
+Hale-1846-179-110	110	Salmon	944	SALMON
+Hale-1846-179-111	111	Sturgeon	3606	STURGEON
+Hale-1846-179-112	112	Name	1405	NAME
+Hale-1846-179-113	113	Affection		
+Hale-1846-179-114	114	White	1335	WHITE
+Hale-1846-179-115	115	Black	163	BLACK
+Hale-1846-179-116	116	Red	156	RED
+Hale-1846-179-117	117	Blue	837	BLUE
+Hale-1846-179-118	118	Yellow	1424	YELLOW
+Hale-1846-179-119	119	Green	1425	GREEN
+Hale-1846-179-120	120	Great	1202	BIG
+Hale-1846-179-121	121	Small	1246	SMALL
+Hale-1846-179-122	122	Strong	785	STRONG
+Hale-1846-179-123	123	Old	1229	OLD
+Hale-1846-179-124	124	Young	1207	YOUNG
+Hale-1846-179-125	125	Good	1035	GOOD
+Hale-1846-179-126	126	Bad	1292	BAD
+Hale-1846-179-127	127	Handsome		
```

Comment:
Just a suggestion, could we map it to BEAUTIFUL? Especially because it stands in opposition to 'ugly'. Besides, BEAUTIFUL refers to appealing appearance properties, judging by its definition.

Reply:
I was wondering the same thing. However, if no one is fully against it, I'd even be for a new concept HANDSOME, because:

- beautiful is 'universal', i.e. applicable to animate and inanimate entities (beautiful landscape, beautiful dog, beautiful human being)
- handsome is (more or less, but more so than beautiful) restricted to human beings
- we already have https://concepticon.clld.org/values/Lynott-2009-423-175 that is unmapped and I've got a another list that also has HANDSOME. :)

Reply:
I'd also agree that it shouldn't be mapped to BEAUTIFUL, because it's less general.

In school we even learned that *handsome* is limited only refer to male humans, but that doesn't seem to be the case.